components/camel-chatscript/src/main/java/org/apache/camel/component/chatscript/ChatScriptBot.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.chatscript;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+import org.apache.camel.component.chatscript.exception.ChatScriptDeliveryException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChatScriptBot {
+
+    private static final transient Logger LOG = LoggerFactory.getLogger(ChatScriptBot.class);
+    String host;
+    int port;
+    String message;
+    String botName;
+    String userName;
+    boolean initialized;
+
+    public ChatScriptBot(String iHost, int port, String iBotName, String iUserName) {
+        this.host = iHost;
+        this.port = port;
+        this.botName = iBotName;
+        this.userName = iUserName;
+    }
+
+    public String sendChat(String input) throws ChatScriptDeliveryException {
```

Comment:
You need to create own exception, when requires to change the default behavior. Please use a standard one for your use-case.

Reply:
modified.